nevergrad/functions/unit_commitment/core.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+import nevergrad as ng
+from ..base import ExperimentFunction
+
+class UnitCommitmentProblem1(ExperimentFunction):
+    """Model that uses conventional implementation for emi-continuous variables
+    The model is adopted from Pyomo model 1: Conventional implementation for emi-continuous variables
+    (https://jckantor.github.io/ND-Pyomo-Cookbook/04.06-Unit-Commitment.html)
+    The constraints are added to the objective with a heavy penalty for violation.
```

Comment:
can you develop what are x and u? if that makes sense, more explicit names would help.